core/fixtures/document-types.json

```diff
@@ -1,27 +1,33 @@
 [
-  {
-    "name": "Other"
-  },
-  
-  {
-    "name": "Contract/Agreement"
-  },
-  {
-    "name": "Consent Form Template"
-  },
-  {
-    "name":  "CNER Approval"
-  },
-  {
-    "name": "CNPD Authorisation"
-  },
-  {
-    "name": "CNPD Notification"
-  },
-  {
-    "name": "ERP Approval"
-  },
-  {
-    "name": "Subject Information Sheet Template"
-  }
+    {
+        "name": "Other"
+    },
+
+    {
+        "name": "DPIA"
```

Comment:
Dear @nene can we replace the short hand with the expanded description i.e.  "Data Protection Impact Asssessment"

Reply:
sure, already replaced